racket/collects/net/http-client.rkt

```diff
@@ -121,34 +129,37 @@
                          #:headers [headers-bs empty]
                          #:content-decode [decodes '(gzip)]
                          #:data [data #f])
+  (define (print-to fmt . args)
+    ;; (eprintf "print-to: ~a~%" (apply format fmt args))
+    (apply fprintf to fmt args))
```

Comment:
I've rolled this back to its original, expanded version

On 09/08/2016 15:45, Jay McCarthy wrote:
> In racket/collects/net/http-client.rkt
> <https://github.com/racket/racket/pull/1411#discussion_r74072424>:
>
>> @@ -121,34 +129,37 @@
>>                           #:headers [headers-bs empty]
>>                           #:content-decode [decodes '(gzip)]
>>                           #:data [data #f])
>> +  (define (print-to fmt . args)
>> +    ;; (eprintf "print-to: ~a~%" (apply format fmt args))
>> +    (apply fprintf to fmt args))
>
> Variable arguments and |apply| cause a lot of allocation and
> uncertainty. A macro version wouldn't have this problem if you wanted to
> do it that way.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/racket/racket/pull/1411/files/a5583485b67cff28edef92607574f1f9288b836c#r74072424>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABDkIaYMp5gfdDzFn2Mkr1j7yaFKDqaNks5qeJJ0gaJpZM4Jf_v6>.
>


-- 
Tim Brown - tim@timb.net - +44(0)7771714159